driver-sync/src/main/com/mongodb/client/FindIterable.java

```diff
@@ -224,4 +224,18 @@
      */
     FindIterable<TResult> showRecordId(boolean showRecordId);
 
+    /**
+     * Enables writing to temporary files on the server. When set to true, the server
+     * can write temporary data to disk while executing the find operation.
+     *
+     * <p>This option is sent only if the caller explicitly provides a value. The default
+     * is to not send a value. For servers &lt; 3.2, this option is ignored and not sent
+     * as allowDiskUse does not exist in the OP_QUERY wire protocol.</p>
```

Comment:
I don't think users will know what `as allowDiskUse does not exist in the OP_QUERY wire protocol` even means.  Consider removing.

Reply:
Done